### PR TITLE
Adds the precision grasper and diamond surgical drill.

### PR DESCRIPTION
**Pull request by DrSnips**
Time to complete the set of advanced surgical tools. (save the fixOvein) Adds in an advanced hemostat called the precision grasper and the diamond surgical drill.
fm
The diamond surgical drill <s>near</s> instantly does it's job and costs diamonds to make and I'm sure this will be used all of 0 times. No other upsides unless you count being kinda expensive an upside.
As for the precision grasper it works twice as fast and when trying to remove something implanted inside someone it guarantees it's retrieval. Costs a bit of plasma, metal, and glass.
Both tools added to the maxed surgeon switchtool to complete it.
:cl:
* rscadd: Added the precision grasper. It functions like a hemostat at double speed and allows you to retrieve implanted things 100% of the time when trying to fish them out.
* rscadd: Added the diamond surgical drill. An advanced version of the surgical drill. It's extremely fast like other diamond drills.

### CODE REVIEW

**Comment by PJB3005** [comment]
Doesn't the incision manager already function as a hemostat + retractor in some cases?

File: code/modules/research/designs/medical.dm [hunk 1]
```diff
@@ -140,6 +150,16 @@
 	category = "Medical"
 	build_path = /obj/item/weapon/circular_saw/plasmasaw
 
+/datum/design/diamond_surgicaldrill
+	name = "Diamond Surgical Drill"
+	desc = "A true extension of the surgeon's body, this marvel instantly cuts the organ, clamp any bleeding, and retract the skin, allowing for the immediate commencement of therapeutic steps."
```
Comment (Pathid) [inline code/modules/research/designs/medical.dm]: Still has the incision manager's `desc`.

**Comment by DamianX** [comment]
yes, for the initial incision, but you can't use it to remove objects

File: code/modules/research/designs/medical.dm (hunk 1 above)
Comment (DrSnips) [reply code/modules/research/designs/medical.dm]: I knew I forgot something in the redo of this. I will fix it.

**Comment by SonixApache** [comment]
weeew

**Comment by ghost** [comment]
That's because it's a retractor, not a hemostat.

**Comment by iFlashYou** [comment]
merge it already

**Pull request merged**